### PR TITLE
=ben fix compilation due to stale PR being merged

### PR DESCRIPTION
**Pull request by ktoso**
Thanks @Hawstein for noticing

### CODE REVIEW

File: akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ class HttpRequestParsingBenchmark {
   implicit val system: ActorSystem = ActorSystem("HttpRequestParsingBenchmark")
   implicit val materializer = ActorMaterializer()
   val parserSettings = ParserSettings(system)
-  val parser = new HttpRequestParser(parserSettings, false, HttpHeaderParser(parserSettings)())
+  val parser = new HttpRequestParser(parserSettings, false, HttpHeaderParser(parserSettings, NoLogging)())
```
Comment (Hawstein) [inline akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala]: Maybe using `system.log` ?

Comment (ktoso) [inline akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala]: No, this is a benchmark - no logging is what we want

Comment (Hawstein) [inline akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala]: Need to import `akka.event.NoLogging`

Comment (ktoso) [inline akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala]: Hah, ofc. thanks for the help @Hawstein.
Rushed fixes are the worst fixes, my bad.

**Comment by ktoso** [comment]
Fixes compilation on master.

**Pull request merged**